pootle/settings.py

```diff
@@ -10,9 +10,78 @@
 import os
 import glob
 
+from django.conf import settings
+from django.core import checks
+
+
 WORKING_DIR = os.path.abspath(os.path.dirname(__file__))
 
 
+DEPRECATIONS = [
+    # old, new, deprecated_version, removed_version
+    ('TITLE', 'POOTLE_TITLE', '2.7', '2.8'),
+    ('CAN_CONTACT', 'POOTLE_CONTACT_ENABLED', '2.7', '2.8'),
+    ('CONTACT_EMAIL', 'POOTLE_CONTACT_EMAIL', '2.7', '2.8'),
+    ('POOTLE_REPORT_STRING_ERRORS_EMAIL', 'POOTLE_CONTACT_REPORT_EMAIL', '2.7', '2.8'),
+    ('PODIRECTORY', 'POOTLE_TRANSLATION_DIRECTORY', '2.7', '2.8'),
+    ('MARKUP_FILTER', 'POOTLE_MARKUP_FILTER', '2.7', '2.8'),
+    ('USE_CAPTCHA', 'POOTLE_CAPTCHA_ENABLED', '2.7', '2.8'),
+    ('POOTLE_TOP_STATS_CACHE_TIMEOUT', None, '2.7', None),
+    ('MT_BACKENDS', 'POOTLE_MT_BACKENDS', '2.7', '2.8'),
+    ('ENABLE_ALT_SRC', None, '2.5', None),
+    ('VCS_DIRECTORY', None, '2.7', None),
+]
+
+
+@checks.register()
+def check_deprecated_settings(app_configs=None, **kwargs):
+    errors = []
+
+    for old, new, dep_ver, remove_ver in DEPRECATIONS:
+
+        # Old setting just disappeared, we just want you to cleanup
+        if hasattr(settings, old) and new is None:
+            errors.append(checks.Info(
+                ("Setting %s was removed in Pootle %s." %
+                 (old, dep_ver)),
+                hint=("Remove %s from your settings." % old),
+                id="pootle.I002",
+            ))
+            continue
+
+        # Both old and new defined, we'd like you to remove the old setting
+        if hasattr(settings, old) and hasattr(settings, new):
+            errors.append(checks.Info(
+                ("Setting %s was replaced by %s in Pootle %s. Both are set." %
+                 (old, new, dep_ver)),
+                hint=("Remove %s from your settings." % old),
+                id="pootle.I002",
+            ))
+            continue
+
+        # Old setting is present and new setting is not defined:
+        # - Warn and copy
+        # - Fail hard if its too old
+        if hasattr(settings, old) and not hasattr(settings, new):
+            from pootle.__version__ import ver
+            if ver >= tuple(remove_ver.split(".")):
```

Comment:
@dwaynebailey Sorry again, just tested your code with `(2, 6, 0) >= tuple('2.6'.split('.'))` and is returning `False`, so it doesn't work as expected. Instead `"2.6.0" >= "2.6"` returns `True` as desired. So please change those two lines of code to use instead what is on the second comment.